src/main/java/org/folio/services/invoice/InvoicePaymentService.java

```diff
@@ -43,6 +44,8 @@ public class InvoicePaymentService {
   private CurrentFiscalYearService currentFiscalYearService;
 
   public static final String INVOICE_LINE_MUST_HAVE_FUND = "The invoice line must contain the fund for payment";
+  public static final Set<CompositePoLine.PaymentStatus> ORDER_LINE_PAYMENT_IGNORED_STATUSES =
+      Set.of(CompositePoLine.PaymentStatus.ONGOING, CompositePoLine.PaymentStatus.PAYMENT_NOT_REQUIRED);
```

Comment:
a bit hard to read, simpler form like this will be more readable: PaymentStatus.ONGOING, PaymentStatus.PAYMENT_NOT_REQUIRED